### PR TITLE
Common search improvements

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes https://github.com/owncloud/web/issues/5644
- Fixes https://github.com/owncloud/web/issues/7587

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by JanAckermann** [comment]
Change log item ist missing and some noise in the unit tests are around

File: packages/web-app-files/src/components/Search/List.vue [hunk 1]
```diff
@@ -1,52 +1,59 @@
...
-    <app-bar :has-bulk-actions="true" />
-    <no-content-message v-if="!paginatedResources.length" class="files-empty" icon="folder">
-      <template #message>
-        <p class="oc-text-muted">
-          <span v-if="!!$route.query.term" v-translate>No resource found</span>
-          <span v-else v-translate>No search term entered</span>
-        </p>
+  <div class="files-search-result oc-flex">
+    <files-view-wrapper>
+      <app-bar :has-bulk-actions="true" :side-bar-open="sideBarOpen" />
```
Comment (kulmann) [inline packages/web-app-files/src/components/Search/List.vue:4]:
```suggestion
      <app-bar :has-bulk-actions="false" :side-bar-open="sideBarOpen" />
```

We don't want to support bulk actions in the search result page at the moment. Please set to `false` :-)

**Review by kulmann — APPROVED** [review]
Minor comment for improvement. Don't know if a changelog item is really needed because the changes were introduced recently that broke the search result page.

**Comment by JanAckermann** [comment]
@kulmann Changelog item is required as it also takes care about the wishes in the linked ticket.

**Pull request merged**